.buildkite/pipeline.yml

```
@@ -41,6 +41,19 @@ steps:
   - group: ":eyes: Special"
     depends_on: "julia"
     steps:
+      - label: "macOS 15"
+        plugins:
+          - JuliaCI/julia#v1:
+              version: "1.10"
+          - JuliaCI/julia-test#v1:
+              test_args: "--quickfail"
+        agents:
+          queue: "juliaecosystem"
+          os: "macos"
+          arch: "aarch64"
+          macos_version: "15.0"
+        if: build.message !~ /\[skip tests\]/ && !build.pull_request.draft
```

I added this before the benchmarks PR was merged so let's make it match the other workflows.
```suggestion
        if: |
          build.message =~ /\[only tests\]/ ||
          build.message =~ /\[only special\]/ ||
          build.message !~ /\[only/ && !build.pull_request.draft &&
            build.message !~ /\[skip tests\]/ &&
            build.message !~ /\[skip special\]/
```